core/src/main/res/values/strings.xml

```diff
@@ -293,6 +293,8 @@
   <string name="grant_read_storage_permission">This app requires the ability to read storage to function. Please grant the permission in your settings</string>
   <string name="go_to_permissions">Go to Settings</string>
   <string name="no_results">No Results</string>
+  <string name="no_bookmarks">No bookmarks yet</string>
```

Comment:
I would remove the "yet"s, it takes a different tone from the rest of our empty messages